Lib/test/test_exceptions.py

```diff
@@ -2387,6 +2387,18 @@ def test_encodings(self):
         finally:
             unlink(TESTFN)
 
+    def test_non_utf8(self):
+        # Check non utf-8 characters
+        try:
+            with open(TESTFN, 'bw') as testfile:
```

Comment:
```suggestion
            with open(TESTFN, 'wb') as testfile:
```
`wb` is more commonly used.